### PR TITLE
return error to client if error happens during update of expression or multientity plugins

### PR DESCRIPTION
**Pull request by AlvaroVega**
I.e. when a InvalidExpression error happens an errror is logged but no response is sent to client

Related with issues https://github.com/telefonicaid/iotagent-node-lib/issues/1175 and https://github.com/telefonicaid/iotagent-node-lib/issues/1223

A group device defined by:

Screenshot from 2022-06-28 11-52-48

And an autoprovisioned device with the following measure:

curl -i -X POST 'http://localhost:7897/iot/json?i=midisp2&k=6idn7z9cba3o57tgoascazdwd' -d '{ "temp": 77}' -H 'content-type: application/json'
HTTP/1.1 400 Bad Request
X-Powered-By: Express
Content-Type: application/json; charset=utf-8
Content-Length: 91
ETag: W/"5b-tRtJaZVLjNLxcjk86ALjp/FAABc"
Date: Tue, 28 Jun 2022 09:30:37 GMT
Connection: keep-alive
Keep-Alive: timeout=5

{"name":"INVALID_EXPRESSION","message":"Invalid expression in evaluation [device:midisp2]"}

Before this PR no response is obtained, but now you got a 400 error with detail.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Fix: return error to client if expresson or multientity plugins gets error
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Should https://github.com/telefonicaid/iotagent-node-lib/issues/1175 or https://github.com/telefonicaid/iotagent-node-lib/issues/1223 or both be mentioned here?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: Just related, but this PR does not fix them

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**